### DISCUSSION TITLE
My mini is only reporting every 1 minute rather than 10 seconds

### DISCUSSION THREAD

**Discussion by blacknell** [BottlecapDave/HomeAssistant-OctopusEnergy#1031] [Q&A] [answered]
Title says it all, but I've seen that it gets data from the meter every 10s but looking at entities in HA it's only doing once per minute.

**Comment by BottlecapDave** [comment, marked as answer]
The integration pulls data at a maximum of once every minute. This is explained in the [docs](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/account/#refresh-rate-in-minutes) and is due to rate limiting on OE side. This could be referenced in the faq for better clarification.

**Reply by blacknell** [reply]
thanks - excellent job BTW.

At times I can't believe all this insight is really possible!
